spec/index.bs

```diff
@@ -1390,10 +1393,6 @@ To get a {{GPUDevice}}, use {{GPUAdapter/requestDevice()}}.
 <script type=idl>
 [Exposed=(Window, DedicatedWorker), Serializable]
 interface GPUDevice : EventTarget {
-    [SameObject] readonly attribute GPUAdapter adapter;
-    readonly attribute FrozenArray<GPUFeatureName> features;
-    readonly attribute object limits;
```

Comment:
This was a way to know the default limits from WebGPU by looking at what's in `device.limits`. How do we envision applications should do this?

Reply:
Good question. Perhaps a default constructor to GPUAdapterFeatures? Or a member on `navigator.gpu`? It would also tell you what limits are understood by the browser without having to request adapters.

Reply:
well, we have the default limits in the spec, wouldn't this make the other means of discovery unnecessary?

Reply:
Other than feature detection, I think so. It would be handy for certain architectures and middlewares but so would a lot of other things we don't provide (like texture size).

Reply:
I guess we could the default limits for all the things in a utility library, it just seems slightly unnecessary.